src/Cargo.lock

```diff
@@ -2167,6 +2337,105 @@ dependencies = [
  "unicode-ident",
 ]
 
+[[package]]
+name = "prometheus"
+version = "0.13.4"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3d33c28a30771f7f96db69893f78b857f7450d7e0237e9c8fc6427a81bae7ed1"
+dependencies = [
+ "cfg-if",
+ "fnv",
+ "lazy_static",
+ "memchr",
+ "parking_lot",
+ "thiserror 1.0.69",
+]
+
+[[package]]
+name = "prost"
```

Comment:
I don't feel great about adding these deps to the QOS workspace since it adds to the dep review burden. Basically any changes to this Cargo.lock incur some overhead and we are adding a bunch of deps to support gRPC/protobuf